src/lv_core/lv_disp.c

```diff
@@ -352,6 +352,21 @@ void lv_disp_trig_activity(lv_disp_t * disp)
     disp->last_activity_time = lv_tick_get();
 }
 
+/**
+ * Invalidate any CPU cache that is related to the display.
+ * @param disp pointer to an display (NULL to use the default display)
+ */
+void lv_disp_invalidate_cache(lv_disp_t * disp)
```

Comment:
For consistence can we name `lv_disp_invalidate_cache` and `clean_dcache_cb` more similarly?

Reply:
I've renamed it to `lv_disp_clean_dcache`.